nextflow/modules/talos/RunHailFiltering/main.nf

```diff
@@ -17,7 +16,7 @@ process RunHailFiltering {
             path("${params.cohort}_small_variants_labelled.vcf.bgz"), \
             path("${params.cohort}_small_variants_labelled.vcf.bgz.tbi")
 
-    // untar the ClinvArbitration data directory
+    // untar the ClinvArbitration data directory. Happy to keep doing this, it's much easier to distribute tar'd
```

Comment:
and it contains some non-MT/HT files, so the compression results in a real-world space saving